client/components/bulk-select/test/index.js

```diff
@@ -43,35 +53,49 @@ describe( 'index', function() {
 		const callback = function() {
 			hasBeenCalled = true;
 		};
-		const bulkSelect = shallow( <BulkSelect selectedElements={ 0 } totalElements={ 3 } onToggle={ callback } /> );
+		const bulkSelect = shallow(
+			<BulkSelect selectedElements={ 0 } totalElements={ 3 } onToggle={ callback } />
+		);
 		bulkSelect.simulate( 'click' );
 		assert.equal( hasBeenCalled, true );
 	} );
 
-	it( 'should be call onToggle with the new state when there are no selected elements', function( done ) {
+	it( 'should be call onToggle with the new state when there are no selected elements', function(
+		done
+	 ) {
```

Comment:
It looks like another bug in the prettier's fork. Eslint says:
> Mixed spaces and tabs  no-mixed-spaces-and-tabs

Somehow it adds space in new line after `done` param.